modules/gdscript/gdscript_compiler.cpp

```diff
@@ -100,6 +100,9 @@ GDScriptDataType GDScriptCompiler::_gdtype_from_datatype(const GDScriptParser::D
 		case GDScriptParser::DataType::BUILTIN: {
 			result.kind = GDScriptDataType::BUILTIN;
 			result.builtin_type = p_datatype.builtin_type;
+			// Fixes GH-87932. Variant::NIL used as default type when getting
+			// the argument type and has_type(false) will be the edge case fix
```

Comment:
```suggestion
			// the argument type and has_type(false) will be the edge case fix.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).